helm/grafana-agent/values.yaml

```diff
@@ -10,6 +10,14 @@ kyvernoPolicyExceptions:
 networkPolicy:
   flavor: cilium
 
+vintageCluster:
+  is: false
```

Comment:
I guess there should be a related PR in `config` to set this value to true on vintage installations/clusters?
This looks very GS-specific though. What is the reason why we need this?
Maybe the name could be more meaningful?

I mean: why do we need this only on vintage MCs and WCs but not on CAPI ones? What is the difference that requires a different behaviour?

Reply:
The difference here is the PSPs that need to be created when running on Vintage